test/e2e/framework/framework.go

```diff
@@ -520,11 +520,6 @@ func (f *Framework) WaitForPodNotFound(podName string, timeout time.Duration) er
 	return e2epod.WaitForPodNotFoundInNamespace(f.ClientSet, podName, f.Namespace.Name, timeout)
 }
 
-// WaitForPodReady waits for the pod to flip to ready in the namespace.
```

Comment:
From the failed tests, I think we may have missed some users of `WaitForPodReady`.

Reply:
you are right.Thanks,i will update it.